decent-dao/fractal-interface
# Sent Asset bug fixes

## Description
Fixed various bugs on the Send Asset Modal:

- removed USD switch from the app as we determine it's usefulness was limited.
- filtered assetsFungible to remove items with zero balance
- do not show Sent Assets button if there are no fungible assets to send
- added toast to Send Assets, to complete this I added toast to useSubmitProposal and then updated all references of that to include appropriate messages.

## Testing

- Verify Toast messages when submitting a new proposal, creating a sub-DAO, and sent assets.
- Verify Send Assets modal work correctly. That the list of assets in the the dropdown only contains items with a balance.
- Verify the Send Assets button does not appear on the Treasury page for a DAO with no assets.
